In [33]:
import pandas as pd


In [32]:
#hier wird eine CSV-Datei geladen
dataframe = pd.read_csv("19-01-02_registrierte_Straftaten_einzeln.csv")
print(dataframe)

      jahr                                           straftat  anzahl
0     2009                       Fahrzeugdiebstahl (Art. 139)    1101
1     2009                                  Betrug (Art. 146)      94
2     2009           Hausfriedensbruch + Diebstahl (Art. 186)     808
3     2009                               Diebstahl (Art. 139)     624
4     2009            Sachbeschädigung + Diebstahl (Art. 144)     782
...    ...                                                ...     ...
2427  2024         Entweichenlassen von Gefangenen (Art. 319)       0
2428  2024         Verletzung des Amtsgeheimnisses (Art. 320)       0
2429  2024  Passive Bestechung schweizerischer Amtsträger ...       0
2430  2024                   Vorteilsannahme (Art. 322sexies)       0
2431  2024  Übertretung firmen- + namensrechtlicher Bestim...       0

[2432 rows x 3 columns]


In [31]:
#so können wir nur einen Teil der Daten auswerten
dataframe = dataframe.loc[dataframe['jahr'] == 2009]

In [27]:
#So können wir den mean (Durchschnitt für alle numerischen Spalten berechnen)
dataframe.mean(numeric_only = True)

jahr      2009.0
anzahl     187.0
dtype: float64

In [23]:
#So können wir den Median berechnen
dataframe.median(numeric_only = True)

jahr      2009.0
anzahl       2.0
dtype: float64

Aufgabe: finde heraus, wie viele "Taschendiebstahl (Art. 139)" es durchschnittlich in den Vergangenen Jahren gab